### ISSUE TITLE
Commit message - interactive link to issue tracker

### ISSUE THREAD

**Issue by smangels** [sublimehq/sublime_merge#557]
**Problem description**
Links to web-based issue trackers are used in software development projects. A user (software developer) to enable a feature rendering these text based tickets IDs (i.e. ARTINFO-12345) into a highlighted interactive link that allows him to immediately browse this issue.

**Preferred solution**

A regular expression in the project-specific configuration (very like the one used in the [Gerrit configuration](https://gerrit-review.googlesource.com/Documentation/config-gerrit.html#commentlink) maybe?).

**Code example**

`
[commentlink "changeid"]
match = (ART[A-Z]+-[0-9]+)
link = "https://jira.example.com/browse/$1"
`

**Comment by srbs** [comment]
duplicate of #42

**Comment by smangels** [comment]
Yes they are related. But what I mean is that there should be code that renders a STRING into an interactive link ARTINFO-12345 into <a href="https://jira.example.com/browse/ARTINFO-12345">ARTINFO-12345</a> and that a user should be able to configure it as described above.

**Comment by natronite** [comment]
It would be nice if `.issuetracker ` files were supported as well.

**Comment by haferburg** [comment]
+1 by me.

For inspiration: https://tortoisegit.org/docs/tortoisegit/tgit-dug-bugtracker.html

> It would be nice if `.issuetracker ` files were supported as well.

@natronite Do you have a link that explains what you mean? What are these files?

**Comment by natronite** [comment]
I found a issues for other clients talking about this:
https://github.com/microsoft/vscode/issues/173000
https://github.com/desktop/desktop/issues/19295

They explain the feature well.
